appendices/spirvenv.txt

```diff
@@ -255,8 +255,10 @@ endif::VK_VERSION_1_1[]
     [eq]#v# components
   * [[VUID-{refpage}-OpTypeImage-04661]]
     Objects of types code:OpTypeImage, code:OpTypeSampler,
-    code:OpTypeSampledImage, and arrays of these types must: not be stored
-    to or modified
+    code:OpTypeSampledImage, code:OpTypeAccelerationStructureKHR,
```

Comment:
The VUID changed here, it refers to OpTypeImage.

Reply:
Sorry, not sure what you mean, I just added `OpTypeAccelerationStructureKHR` from the other VU to here

Reply:
ITHKM that the VUID should be regenerated since the semantics of the VU statement changed - but since these are usually VUs you yourself are implementing, often that doesn't matter.

Reply:
just removed the VUID, its just a number and I don't really care what it is, just that there is one to track